File: client/src/app/game.ts
export interface Game {
  _id: string;
  //prompt?: string;
  players?: string[];
  responses?: string[];
  judge?: number;
  scores?: number[];
  winnerBecomesJudge?: boolean;
  pastResponses?: string[];
  winningScore?: number;
  winner?: string; // Add a property to store the winner's name
}

File: server/src/main/java/umm3601/game/Game.java
package umm3601.game;

import org.mongojack.Id;
import org.mongojack.ObjectId;

@SuppressWarnings({"VisibilityModifier"})
public class Game {
  @ObjectId @Id
  @SuppressWarnings({"MemberName"})
  public String _id;

  public String[] players;
  public String[] responses;
  public int judge;
  public int[] scores;
  public boolean discardLast;
  public boolean winnerBecomesJudge;
  public String[] pastResponses;
  public int winningScore;
  public String winner; // Add a field to store the winner's name
}
